content/en/docs/performance/resource_consumption.md

```diff
@@ -0,0 +1,122 @@
+---
+draft: false
+linktitle: Kmesh resource consumption
+menu:
+  docs:
+    parent: performance
+    weight: 2
+title: Kmesh resource consumption
+toc: true
+type: docs
+---
+# 1. Environment setup
+![resource_env](images/resource_test_env.png)
+
+- K8S version: v1.27
+- Kmesh version: 0.4
+- Kernel version: 5.10
+- Node flavor: 8U16G
+- testing tool: fortio
+- metric collecting tools: top, bpftop
+
+
```

Comment:
I think it's necessary to add an overall introduction to the purpose of this article